code/web/src/modules/user/api/actions.js

```diff
@@ -36,7 +36,7 @@ export function login(userCredentials, isLoading = true) {
     return axios.post(routeApi, query({
       operation: 'userLogin',
       variables: userCredentials,
-      fields: ['user {name, email, description, shippingAddress, shipDate, imageUrl, role, }', 'token']
+      fields: ['user {name, email, role, }', 'token']
```

Comment:
http://gph.is/1aR7NWq

Reply:
Gotta love how one small thing can break everything.